LedgerHQ/app-xrp
# App update

Update the repository:
- Change `Makefile` to default Ledger standard
- Fix deprecated API
- Remove unneeded `glyphs`, already present in SDK
- Add **nanosp** support, with corresponding screenshots
- Add the missing Ledger CI workflows and adapt current ones
- Fix Linters output
- Add missing `ragger` tests
  - Add `utils` module with generic `navigate` function
  - Update `get_public_key`
    - Fix length and check returned values (including `chain_code`)
    - Add new tests _accepted_ and _rejected_ with the  screenshots
  - Add `get_configuration` test to check the returned version
  - Update `sign_valid_tx`
    - Add signature check
    - Add new test _rejected_ with the  screenshots
- Fix minor documentation formatting
- Fix decimal parsing for special encoding for 0 from old PR (see https://github.com/LedgerHQ/app-xrp/pull/23)
- Update version (2.3.1 to 2.3.2)
